src/geckomat/utilities/enzymeUsage.m

```diff
@@ -0,0 +1,49 @@
+function [absUsage,capUsage,UB,protID] = enzymeUsage(ecModel,fluxes,zero)
```

Comment:
Do these functions support the light models? If not, at least write that in the header. It shouldn't be that hard to implement I guess.

Reply:
Not supported, as it currently relies on the usage_prot reactions. I'll mention this for the moment, and in the long run it can indeed be implemented for geckoLight as well.